src/libraries/Microsoft.Extensions.Options.ConfigurationExtensions/tests/Common/OptionsBuilderConfigurationExtensionsTests.cs

```diff
@@ -170,8 +171,8 @@ public static void BindConfiguration_UpdatesOptionOnConfigurationUpdate()
             services.AddSingleton<IConfiguration>(new ConfigurationBuilder()
                     .Add(configSource)
                     .Build());
-            OptionsBuilder<FakeOptions> optionsBuilder = services.AddOptions<FakeOptions>();
-            _ = optionsBuilder.BindConfiguration(configSectionName);
+            _ = services.AddOptions<FakeOptions>()
+                .BindConfiguration(configSectionName);
```

Comment:
If I understand correctly, the newline is an important part of this test case. If that's right, then we should add some generous code comments around this to make sure the code formatting is preserved over time and no one inadvertently combines the lines together. If the test does have the purpose of verifying by syntax, then the test name should also reflect that. With this PR and with other fixes related to syntax / code formatting, we should be sure code comments and test names make such coverage notes very clear.

If a change is made here, please make the change first in main and then cherry-pick that merged commit into this branch/PR as well.

Reply:
Indeed, the newline is important. I'll revert the opportunistic diff here & add a dedicated test suite to the project.

Reply:
We're discussing this in https://github.com/dotnet/runtime/pull/91237.